infra/src/main/kotlin/fi/fta/geoviite/infra/projektivelho/PVApi.kt

```diff
@@ -159,9 +159,9 @@ data class PVApiFileMetadata(
     @JsonProperty("sisaltaa-henkilotietoja") val containsPersonalInfo: Boolean?
 )
 
-data class PVApiFile(
+data class PVApiDocument(
```

Comment:
Nämä nimeämiset voisi ehkä käydä vielä jossain kohtaa läpi (esim. jonkun dailyn yhteydessä.) Mulla on sellainen fiilis, että näissä on nyt vähän jotain epäsynkkaa meidän sisäisten, API-tietotyyppien ja kantataulujen nimeämisten välillä, mutta en jotenkin osaa suoraan pistää sanoiksi että minkälaista.

Reply:
Joo käydään toki. Siksi näitä siis tein että siellä oli samalle asialle file/document ristiin. Nyt sen pitäisi olla aika lailla document (mikä on velhon käyttämä termi). Frontissa voi olla vielä vanhoja nimiä.